### PR TITLE
Added Featured servers to server list

### PR DESCRIPTION
**Pull request by Greaper88**

### CODE REVIEW

**Comment by lukeeey** [comment]
Why?

**Comment by rtm516** [comment]
It made sense to change it, I asked him to PR it

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -44,16 +44,56 @@ geyser:
 
 # A global list of servers sent to all clients
 servers:
-  - address: test.geysermc.org
+  - name: The Hive
+    address: 167.114.81.89
```
Comment (lukeeey) [inline src/main/resources/config.yml:48]: `hivebedrock.network`

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -44,16 +44,56 @@ geyser:
...
-    name: Official Geyser Test Server
-    imageUrl: https://geysermc.org/android-chrome-192x192.png
     category: OFFICIAL
-  - address: "play.cubecraft.net"
-  - address: "127.0.0.1"
-    port: 25565
+    imageUrl: 'https://i.imgur.com/VQJW6XG.png'
+
+  - name: CubeCraft
+    address: 94.23.159.81
```
Comment (lukeeey) [inline src/main/resources/config.yml:55]: `mco.cubecraft.net`

File: src/main/resources/config.yml [hunk 3]
```diff
@@ -44,16 +44,56 @@ geyser:
...
+  - name: Galaxite
+    address: 54.39.243.199
+    port: 19132
+    bedrock: true
+    category: OFFICIAL
+    imageUrl: >-
+      https://pbs.twimg.com/profile_images/1275867042583896066/UMPF5nTM_400x400.jpg
+
+  - name: Lifeboat Network
+    address: 63.143.54.198
```
Comment (lukeeey) [inline src/main/resources/config.yml:70]: `mco.lbsg.net`

File: src/main/resources/config.yml [hunk 4]
```diff
@@ -44,16 +44,56 @@ geyser:
...
+    port: 19132
+    bedrock: true
+    category: OFFICIAL
+    imageUrl: 'https://lbsg.net/wp-content/uploads/2017/06/lifeboat-square.png'
+
+  - name: Mineplex
+    address: 108.178.12.38
```
Comment (lukeeey) [inline src/main/resources/config.yml:77]: `mco.mineplex.com`

File: src/main/resources/config.yml [hunk 5]
```diff
@@ -44,16 +44,56 @@ geyser:
...
+    port: 19132
+    bedrock: true
+    category: OFFICIAL
+    imageUrl: 'https://www.mineplex.com/assets/www-mp/img/footer/footer_smalllogo.png'
+
+  - name: MineVille
+    address: 52.234.130.255
```
Comment (lukeeey) [inline src/main/resources/config.yml:84]: `play.inpvp.net`

File: src/main/resources/config.yml [hunk 6]
```diff
@@ -44,16 +44,56 @@ geyser:
...
+
+  - name: CubeCraft
+    address: 94.23.159.81
+    port: 19132
+    bedrock: true
+    category: OFFICIAL
+    imageUrl: 'https://www.cubecraft.net/attachments/q3pmrsod_400x400-png.53219/'
+
+  - name: Galaxite
+    address: 54.39.243.199
```
Comment (lukeeey) [inline src/main/resources/config.yml:62]: `play.galaxite.net`

**Review by lukeeey — CHANGES_REQUESTED** [review]

**Pull request merged**